remoteappmanager/docker/container_manager.py

```diff
@@ -165,7 +158,7 @@ def _start_container(self, user_name, image_name, volumes):
         # build the dictionary of keyword arguments for create_container
         container_name = _generate_container_name("remoteexec",
                                                   user_name,
-                                                  image_name)
+                                                  mapping_id)
```

Comment:
It seems that there is an inconsistency of my memory and your memory of what the mapping id is used for.  According to my problematic memory, the mapping id is a unique identifier for application+policy (or just policy?) so that, for example if the policy says only 10 instances is allowed, we make sure there are no more than 10 containers with this mapping id.  Nothing is said about how these instances are shared among users.